views/doc.cfm

```diff
@@ -96,7 +96,10 @@
 				<li role="separator" class="pull-right divider"></li>
 			</cfif>
 			<li class="pull-right">
-				<a href="https://github.com/foundeo/cfdocs/issues/new?title=#encodeForURL(data.name)#" rel="nofollow" class="label label-warning" title="Report an Issue">Issue</a>
+		    		<span class="label label-warning">
+					<a href="https://github.com/foundeo/cfdocs/issues?q=is:issue%20is:open%20#encodeForURL(data.name)#" class="issuecount badge">4</a>
```

Comment:
Was talking about the `4` you can see here... the issue count will be 4 until the JS executes.

Reply:
Oh, I totally forgot to delete it. Maybe the badge should be hidden when no issues are found?